test/Microsoft.SqlTools.ServiceLayer.PerfTests/Microsoft.SqlTools.ServiceLayer.PerfTests.xproj

```diff
@@ -7,7 +7,7 @@
   <Import Project="$(VSToolsPath)\DotNet\Microsoft.DotNet.Props" Condition="'$(VSToolsPath)' != ''" />
   <PropertyGroup Label="Globals">
     <ProjectGuid>7e5968ab-83d7-4738-85a2-416a50f13d2f</ProjectGuid>
-    <RootNamespace>Microsoft.SqlTools.ServiceLayer.PerfTests</RootNamespace>
+    <RootNamespace>Microsoft.SqlTools.ServiceLayer.PerfTests.Tests</RootNamespace>
```

Comment:
The root namespace should be without the .Tests, otherwise anything in the tests folder should technically be "Microsoft.SqlTools.ServiceLayer.PerfTests.Tests.Tests". It's not a huge issue, but without it, we'll get warnings from ReSharper. If there's a functional reason for the change, then that's fine.